doorbell/doorbell_backend/src/logupdate.py

```diff
@@ -1,31 +1,81 @@
-import os
-from datetime import datetime
 import csv
-import time
-import re
-from pathlib import Path
+from datetime import datetime
+from time import ctime
+from re import search, sub
+from os.path import isfile, join
+from json import load, dump
+
+import connect
+
+import constants
+
+
+def updateStatus(update_status):
+    with open(constants.CONNECT_SETTINGS_PATH, "r") as jsonFile:
+        data = load(jsonFile)
+        if (
+            data["Connection Settings"][3]["Connection Status"] == "False"
+            and update_status == "True"
+        ):
+            sendBuffer()
+        data["Connection Settings"][3]["Connection Status"] = update_status
+        print(data)
+    with open(constants.CONNECT_SETTINGS_PATH, "w") as jsonFile:
+        dump(data, jsonFile, indent=4)
 
 
-def updateLogs(log_path, filename, status):
-    log_file = Path(log_path + "/storage_info.csv")
+def updateLogs(filename, status):
+    createLogs()
+    # Write to log file
+    with open(constants.LOG_FILE_PATH, mode="a", newline="") as storage_info:
+        storage_info_writer = csv.writer(
+            storage_info, delimiter=",", quoting=csv.QUOTE_MINIMAL
+        )
+        local_time = ctime(int(filename.split(".")[0]))
+        time_to_store = search(constants.TIME_REGEX, local_time).group(0)[:-1]
+        date_to_store = sub(constants.TIME_REGEX, "", local_time)
+        storage_info_writer.writerow(
+            [filename[:-4], date_to_store, time_to_store, status]
+        )
 
-    if not log_file.is_file():
-        with open(log_path + "/storage_info.csv", mode="w", newline="") as storage_info:
+
+# Creates the log file if not exists with headers
+def createLogs():
+    if not isfile(constants.LOG_FILE_PATH):
+        with open(constants.LOG_FILE_PATH, mode="w", newline="") as storage_info:
             storage_info_writer = csv.writer(
-                storage_info, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL
+                storage_info, delimiter=",", quoting=csv.QUOTE_MINIMAL
             )
             storage_info_writer.writerow(["Filename", "Date", "Time", "Status"])
```

Comment:
Can add this list to constants.py

Reply:
Maybe, but it wouldn't save that much room as a constant IMO.